src/components/layout/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,282 @@
+  color: ${({ level }) => (level === 1 ? color.slate800 : color.slate500)};
+
+  a {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 14px;
+    text-decoration: none;
+    transition: color 0.1s ease-in-out;
+    color: inherit;
+
+    &:hover {
+      color: ${color.slate800};
+    }
+  }
+`;
+
+const NavAccordionTrigger = styled(Accordion.Trigger)`
+  all: unset;
+  display: flex;
+  height: 28px;
+  ${typography.body14}
+  color: ${color.slate800};
+  transition: color 0.1s ease-in-out;
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+  color: ${color.slate500};
+
+  svg {
+    transition: transform 0.2s ease-in-out;
+  }
+
+  &:hover {
+    color: ${color.slate800};
+  }
+
+  &[data-state='open'] {
+    svg {
+      transform: rotate(90deg);
+    }
+  }
+`;
+
+const slideDown = keyframes`
+  from {
+    height: 0;
+  }
+  to {
+    height: var(--radix-accordion-content-height);
+  }
+`;
+
+const slideUp = keyframes`
+  from {
+    height: var(--radix-accordion-content-height);
+  }
+  to {
+    height: 0;
+  }
+`;
+
+const NavAccordionContent = styled(Accordion.Content)`
+  overflow: hidden;
+
+  &[data-state='open'] {
+    animation: ${slideDown} 300ms cubic-bezier(0.87, 0, 0.13, 1);
+  }
+  &[data-state='closed'] {
+    animation: ${slideUp} 300ms cubic-bezier(0.87, 0, 0.13, 1);
+  }
+`;
+
+export const Sidebar: FC<SidebarProps> = ({
+  docsTocWithLinkWrappers,
+  versions: versionsProp,
+  slug,
+}) => {
+  const { isLatest, version, versionString } = useSiteMetadata();
+
+  const versions = versionsProp || {
+    stable: [
+      {
+        version,
+        string: versionString,
+        label: isLatest ? 'Latest' : undefined,
+      },
+    ],
+    preRelease: [],
+  };
+
+  return (
+    <div>
+      <nav>
+        <TopNav>
+          <Line>
+            <Link to="/docs">
+              <DocsIcon />
+              Documentation
+            </Link>
+          </Line>
+          <Line>
+            <Link to="/tutorials">
+              <TutorialsIcon />
+              Tutorials
+            </Link>
+          </Line>
+          <Line>
+            <Link to="/releases">
+              <ChangelogIcon />
+              Changelog
+            </Link>
+          </Line>
+          {/* <Line>
+            <Link to="/docs/api">
+              <APIIcon />
+              API
+            </Link>
+          </Line> */}
+        </TopNav>
+      </nav>
+      <VersionSelector version={version} versions={versions} slug={slug} />
+      <Accordion.Root type="multiple" asChild>
+        <AccordionRoot>
+          {/* eslint-disable react/no-array-index-key */}
+          {docsTocWithLinkWrappers.map((lvl1, lvl1Index) => (
+            <Fragment key={lvl1Index}>
+              <NavItem level={1}>
+                {lvl1.type === 'heading' ? <a href={lvl1.pathSegment}>{lvl1.title}</a> : lvl1.title}
```

Comment:
Without this, the current top-level items that aren't headings (FAQ and Migrate to 7.0) do not render as links.

```suggestion
                { ['link', 'heading'].includes(lvl1.type) ? <a href={lvl1.pathSegment}>{lvl1.title}</a> : lvl1.title}
```